dlite_cuds/strategies/parse.py

```diff
@@ -81,16 +81,29 @@ def get(
         Returns:
             key to CUDS object in cache.
         """
-        cache = DataCache(self.parse_config.configuration.datacache_config)
-
+        # cache = DataCache(self.parse_config.configuration.datacache_config)
```

Comment:
Maybe add a comment about why you have commented out creating the cache with the datacache_config arguments.